sklearn/metrics/_plot/confusion_matrix.py

```diff
@@ -89,14 +89,14 @@ def plot(self, include_values=True, cmap='viridis',
         if include_values:
             self.text_ = np.empty_like(cm, dtype=object)
             if values_format is None:
-                values_format = '.2g'
+                values_format = lambda x: '.2g' if (log10(x) > 7) else 'd'
```

Comment:
this should just be a strong, you can remove the lambda

Reply:
Will do ✔

Reply:
Hey Nicolas, I'm unsure how to implement this since the values only get passed in the for loop that's ran after this code. The values are then cm[i, j], where we should check these values for log10(cm[i,j]) < 7.